metricbeat/module/kubernetes/state_replicaset/data.go

```diff
@@ -0,0 +1,59 @@
+package state_replicaset
```

Comment:
I see the point. Leads me to think it should be it's own module but we concluded this discussion :-) Don't have a better solution at the moment :-(